packages/plugin-packages/mashroom-portal-sandbox-app/src/js/components/MessageBusSendForm.tsx

```diff
@@ -81,7 +87,8 @@ export default class MessageBusSendForm extends PureComponent<Props> {
                         <SelectField id='mashroom-sandbox-publish-message-topic' name='topic' labelId='topic' options={topicOptions} emptyOption={true} />
                     </div>
                     <div className='mashroom-sandbox-app-form-row'>
-                        <SourceCodeEditorField id='mashroom-sandbox-publish-message-message' name="message" labelId='message' language='json' theme='light' height={120} />
+                        { this.sbAutoTest === 0 && <SourceCodeEditorField id='mashroom-sandbox-publish-message-message' name="message" labelId='message' language='json' theme='light' height={120} /> }
```

Comment:
no good idea to test for 0 or 1 - please just test if the sbAutoTest param exists or not